### ISSUE TITLE
Freeze does not seem to do anything

### ISSUE THREAD

**Issue by mmaimer33** [mmaimer33/ped#5]
Running the `freeze` command when listing, finding, filtering (both before and after) does not seem to affect the functioning of the app in any way. E.g.
With persons called "Test 1", "Test 2", and "Test 3" and one called "Person 2", run
1. `filter n/Test`
2. `freeze`
3. `filter n/2`
--> This seems to list "Test 2" and "Person 2" regardless of whether the address book was frozen.
4. `unfreeze` --> This gives the error "Address book is not frozen!"
Note: the error is separate from the success message of `unfreeze`, "Address book has been unfrozen!"

Basically, as a new user, I have no idea what this command does.
